client/my-sites/google-my-business/stats/chart.js

```diff
@@ -24,70 +25,97 @@ import { getGoogleMyBusinessStats } from 'state/selectors';
 import { getInterval } from 'state/google-my-business/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+function transformData( props ) {
+	const { data } = props;
+
+	if ( ! data ) {
+		return data;
+	}
+
+	const aggregation = getAggregation( props );
+
+	if ( aggregation === 'total' ) {
```

Comment:
Note: this is modified in https://github.com/Automattic/wp-calypso/pull/24236 as well